MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -147,15 +152,38 @@ object VisionManager {
         this.application = application
     }
 
-    /**
-     * Initialize SDK. Creates core services and allocates necessary resources.
-     * No-op if called while SDK is created already.
-     */
-    @JvmOverloads
-    @JvmStatic
-    fun create(
-        videoSource: VideoSource = Camera2VideoSourceImpl(application)
-    ) {
+    sealed class VisionMode {
```

Comment:
may be better to move sealed class out of outer class

Reply:
Agree, will do after POC, need to arrange final API look before changes